.bmad/bmb/workflows/create-agent/agent-command-patterns.md

```
@@ -0,0 +1,759 @@
+    <item cmd="*interpret">Interpret results</item>
+
+    <item cmd="*exit">Exit with confirmation</item>
+  </menu>
+</agent>
+```
+
+## LLM Building Guide
+
+When creating commands:
+
+1. Start with *help and *exit
```

markdownlint-cli2 (0.18.1)

Spaces inside emphasis markers

(MD037, no-space-in-emphasis)